src/components/content/TextAreaComponent.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Box, Textarea } from "@chakra-ui/react";
+
+const TextAreaComponent: React.FC = (props) => {
+  return (
+    <Box my={5}>
+      <Textarea
+        borderWidth={0}
```

Comment:
Isn't borderWidth equal to zero by default?

Reply:
no in Textarea